### ISSUE TITLE
Review desktop app wrapper

### ISSUE THREAD

**Issue by eliselavy** [C2DH/jdh-notebook#34]
https://blog.jupyter.org/jupyterlab-desktop-app-now-available-b8b661b17e9a

**Comment by inactinique** [comment]
Nope, no way. It implies to install conda or miniconda, which is far more of a problem than installing jupyter lab or notebooks and start them on CLI.
We can mention for those who are indeed using conda that they can use it, with the problem of cite2c (there is a version that works with lab, but not supposed to be used yet for production).
